lib/vanagon/platform/rpm.rb

```diff
@@ -45,15 +45,15 @@ def package_name(project)
         "#{project.name}-#{project.version}-#{project.release}.#{project.noarch ? 'noarch' : @architecture}.rpm"
       end
 
-      def output_dir(target_repo = "products")
-        super
-      end
```

Comment:
BUT WORTH IT TO FIX. It looks like that's mainly going to be a pain with promotions. We'll have to coordinate this and the fix to the promotion code in enterprise-dist